src/options/ui.js

```diff
@@ -187,6 +189,8 @@ const loadFromFile = async ({ currentTarget }) => {
   } catch (exception) {
     window.alert(exception.toString());
     currentTarget.value = currentTarget.defaultValue;
+  } finally {
+    currentTarget.value = '';
```

Comment:
No need to have two lines doing the same thing!

```suggestion
    window.alert(exception.toString());
  } finally {
    currentTarget.value = currentTarget.defaultValue;
```

Reply:
Whoa, I completely missed that line existing 😆